.github/workflows/deploy.yml

```diff
@@ -27,7 +23,7 @@ jobs:
         run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
       
       - name: Update release files
-        run: cd src/ontology/ && docker run -v $PWD/../../:/work -w /work/src/ontology -e ROBOT_JAVA_ARGS='-Xmx10G' -e JAVA_OPTS='-Xmx10G' --rm obolibrary/odkfull make GITHUB_ACTION=true prepare_release_fast
+        run: cd src/ontology/ && make ROBOT_ENV='ROBOT_JAVA_ARGS=-Xmx10G' GITHUB_ACTION=true prepare_release_fast
```

Comment:
@anitacaron do you understand why it says

```
ROBOT_ENV='ROBOT_JAVA_ARGS=-Xmx10G'
```

instead of 

```
ROBOT_JAVA_ARGS=-Xmx10G
```

? I can see this also exists in other pipelines but I cannot remember putting this anywhere.

Reply:
I don't remember this either. I've checked the commit history and couldn't find anything.